client/src/components/Join.jsx

```diff
@@ -19,7 +19,7 @@ const Join = ({ setTopLevelState }) => {
 
   return (
     <form>
-      <label forHtml='login'>Add your name</label>
+      <label forHtml='login'> Login As A Guest </label>
```

Comment:
Not a big deal, but FYI the single spaces preceding/succeeding the text is rendered.

Reply:
The spaces are outta there